### ISSUE TITLE
Some mob's loot hitboxes are off

### ISSUE THREAD

**Issue by aajonusonline** [Globox1997/RpgZ#1]
This is not the full list, there is more.
![image](https://user-images.githubusercontent.com/62964550/101620932-cb0cac00-3a1d-11eb-81b6-6a1c0fbb4501.png)
![image](https://user-images.githubusercontent.com/62964550/101621015-e8da1100-3a1d-11eb-9efa-5184fd89190e.png)

**Comment by Globox1997** [comment]
Yes that's right, the hitboxes aren't working 100% perfect but that's the best way how I got it. Tried different things and that was the best solution.

**Comment by Globox1997** [comment]
Would be awesome if you improve it :)

**Comment by Globox1997** [comment]
are there any hitboxes which are far off actually?
